README.zh-CN.md

```diff
@@ -43,6 +43,7 @@
 -   `callback` ：完成后执行的回调函数；
 -   `stream` 终端中的输出格式流，默认 `stderr` ；
 -   `head` ：进度条的头字符；
+-   `srcDir`
```

Comment:
I will need some help with the translation here.  Sorry :(

Reply:
It doesn't matter. I'll add it in Chinese.